web/src/components/modals/AutomaticUpdatesModal.tsx

```diff
@@ -312,10 +339,9 @@ export default class AutomaticUpdatesModal extends React.Component {
                 isSearchable={false}
                 getOptionValue={(schedule) => schedule.label}
                 value={selectedSchedule}
+                // TODO: upgrade react-select and fix this
+                // @ts-ignore
                 onChange={this.handleScheduleChange}
-                isOptionSelected={(option) => {
```

Comment:
this was never true, selectedSchedule is an object not string. 
I tried fixing this by setting `option.value === selectedSchedule.value`, however it messed up the hover states on select element. I removed it and nothing changed.